app/celeryapp/__init__.py

```diff
@@ -0,0 +1,27 @@
+from celery import Celery
+
+celery = None
```

Comment:
where are we using this variable?

Reply:
This variable was being used as a placeholder(I don't know a better term) for the celery app which was being initialized in the `create_app` function and then this variable would be updated to represent the created celery app. I have removed it since putting the Celery configs into the base app makes the variable and initializing the app in `create_app` not necessary. I tested the Celery app with the new configs and is working fine. This also resolves the import issues as well.